### PR TITLE
add --split to specify train/test/dev dataset

### PR DESCRIPTION
**Pull request by MintForever**
## Changes
-attack_args_helpers.py has been modified to add the '--split' flag and an 'args.split' check 
-huggingface_dataset.py has been modified to change the default value for 'split' from 'train' to 'test'

### CODE REVIEW

**Comment by qiyanjun** [comment]
@MintForever use black to fix the formatting issue..

**Comment by qiyanjun** [comment]
@willyptrain please help review this!

**Comment by qiyanjun** [comment]
@tahmid-kazi please help review this

**Comment by qiyanjun** [comment]
@cogeid  please add your comment!

**Review by cogeid — APPROVED** [review]
Passed:

Attack with SNLI dataset
Attack with rotten tomatoes dataset
Attack with IMBD dataset

File: tests/test_command_line/test_attack.py [hunk 1]
```diff
@@ -199,6 +203,9 @@ def test_command_line_attack(name, command, sample_output_file):
 
     if DEBUG and not re.match(desired_re, stdout, flags=re.S):
         pdb.set_trace()
+    print("??????????desired_re", desired_re)
+    print("??????????desired_re", stdout)
```
[the remark this answers is not in the document]
Comment (qiyanjun) [reply tests/test_command_line/test_attack.py:207]: @MintForever please respond to the above comment by @cogeid

[the remark this answers is not in the document]
Comment (qiyanjun) [reply tests/test_command_line/test_attack.py:207]: @MintForever can you solve Cogeid's concern?

**Review by jinyongyoo — APPROVED** [review]
LGTM!

**Pull request merged**